src/Stripe.net/Entities/SubscriptionSchedules/SubscriptionSchedulePhasePlan.cs

```diff
@@ -0,0 +1,60 @@
+namespace Stripe
+{
```

Comment:
you forgot to remove this class which shouldn't be codegen-ed anymore

Reply:
Nice catch. Thanks!